programs/util/sign_transaction.cpp

```diff
@@ -49,7 +51,7 @@ int main(int argc, char** argv, char** envp)
       tx_signing_result sres;
       sres.tx = sreq.tx;
       sres.digest = sreq.tx.digest();
-      sres.sig_digest = sreq.tx.sig_digest(STEEM_CHAIN_ID);
+      sres.sig_digest = sreq.tx.sig_digest( _chain_id );
```

Comment:
When you revert the chain id name change, this file should be unchanged.

Reply:
Done